pixl_dcmd/pyproject.toml

```diff
@@ -16,6 +16,7 @@ dependencies = [
     "pydicom-data",
     "python-decouple==3.8",
     "requests==2.32.3",
+    "setuptools", # pydicom needs this
```

Comment:
I deleted this dependency and tried the following:
```python
uv python install 3.11
uv venv --python 3.11
source .venv/bin/activate
uv sync
```
and it seemed to install okay. What was the error you ran into?

Reply:
I think this will give an error in the system tests when orthanc tries to run or load plugins.

Reply:
I tested it and the system test works fine without it. Not sure why it fixed something at the time!

Reply:
Ah sorry I misremembered, it was the pixl_dcmd test that fails without it.